### ISSUE TITLE
DMC4 Crashes On Launch

### ISSUE THREAD

**Issue by OmeggaN** [serpentiem/ddmk#311]
Don't know what to do, my game just launch and crash just after, help pls

**Comment by SteveCraft1** [comment]
This Lady is not finished one from the special edition game(you can even access it on PS2 by gameshark code),try using Boss Lady instead

**Comment by OmeggaN** [comment]
Yeah, boss version works pretty fine actually, but i'll wait for our best boy to fix the normal one o/

**Comment by SteveCraft1** [comment]
Then its gonna be forever because serpentiem is already done with ddmk so theres no more update in it

**Comment by SilverLeg** [comment]
> Автор

Choose the basis of the Dante model (or Virgil, depending on what techniques and weapons you want), and below the appearance of the Lady.
